src/commands/deployment/helper.ts

```diff
@@ -0,0 +1,221 @@
+import Table from "cli-table";
+import * as fs from "fs";
+import * as os from "os";
+import Deployment from "spektate/lib/Deployment";
+import AzureDevOpsPipeline from "spektate/lib/pipeline/AzureDevOpsPipeline";
+import IPipeline from "spektate/lib/pipeline/Pipeline";
+import { logger } from "../../logger";
+
+let hldPipeline: IPipeline;
+let clusterPipeline: IPipeline;
+let srcPipeline: IPipeline;
+const fileLocation = os.homedir() + "/.Spektate";
+export let config: { [id: string]: string } = {};
+export enum OUTPUT_FORMAT {
+  NORMAL = 0,
+  WIDE = 1,
+  JSON = 2
+}
+
+export class Helper {
```

Comment:
It would be good to have all exported/public classes/functions/methods to have some accompanying jsdoc